spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -1309,42 +1424,42 @@ dataflow:>task execution list
 ║task1    │1 │Fri May 05 18:12:05 EDT 2017│Fri May 05 18:12:05 EDT 2017│0        ║
 ╚═════════╧══╧════════════════════════════╧════════════════════════════╧═════════╝
 ----
+====
 
-
-2) Destroy the task, by using the following command:
-
-
+. Destroy the task, by using the following command:
++
+====
 [source,bash]
 ----
 dataflow:>task destroy --name task1
 ----
-
+====
 
 === Application and Server Properties
 
 This section covers how you can customize the deployment of your applications. You can use a number of properties to influence settings for the applications that are deployed. Properties can be applied on a per-application basis or in the appropriate server configuration for all deployed applications.
 
-NOTE: Properties set on a per-application basis always take precedence over properties set as the server configuration. This arrangement allows for the ability to override global server level properties on a per-application basis.
+NOTE: Properties set on a per-application basis always take precedence over properties set as the server configuration. This arrangement lets you override global server level properties on a per-application basis.
 
-Properties to be applied for all deployed Tasks are defined in the `src/kubernetes/server/server-config-(binder).yaml` file and for Streams in `src/kubernetes/skipper/skipper-config-(binder).yaml`. Replace `(binder)` with the messaging middleware you are using, for example `rabbit` or `kafka`.
+Properties to be applied for all deployed Tasks are defined in the `src/kubernetes/server/server-config-(binder).yaml` file and for Streams in `src/kubernetes/skipper/skipper-config-(binder).yaml`. Replace `(binder)` with the messaging middleware you are using -- for example, `rabbit` or `kafka`.
```

Comment:
Not sure I follow this pattern: `.. using -- for example ...`

Reply:
"for example whatever" is incorrect. There must be a comma after "for example". However, that makes for a clutter of commas, so I change the comma after "using" to an em dash (the two hyphens get rendered as an em dash when Asciidoctor does its thing).

The deeper issue is that the sentence has both the instruction and an example in it. The dash separates the two parts, and the comma properly separates "for example" (because it's parenthetical to the main content).